spec/connect/client_spec.rb

```diff
@@ -343,8 +343,8 @@
       before { recommended_2.available = false }
 
       it 'does not register the unavailable extension nor its children' do
-        expect(subject).not_to receive(:register_product).with(recommended_2)
-        expect(subject).not_to receive(:register_product).with(recommended_2_2)
+        expect(subject).not_to receive(:register_product).with(recommended_2, any_args)
```

Comment:
the `any_args` should not be needed anymore right?

Reply:
Yes but why adding it if there is no need for it. Also it adds the possibility to execute `register_product` with `any_args`.

Reply:
True. Fixed.